### DISCUSSION TITLE
Adding anchor link to header

### DISCUSSION THREAD

**Discussion by TristanCacqueray** [srid/emanote#453] [Ideas]
Hello,

How to make header title linkable, e.g. by having a clickable chain-icon before the title, similar to what GitHub do?
Is there a ready to use pandoc filter or something we can enable somewhere?

Thanks!

**Comment by kukimik** [comment]
This was proposed in https://github.com/srid/emanote/discussions/137 (and the proposal was accepted). I wanted to work on this, but never found the time. Maybe I'll manage to get back to it.
